Support for React 15.4.x?

It looks like this library has been broken by the move to React 15.4.x.

````
ERROR in ./~/html-react-parser/lib/attributes-to-props.js
Module not found: Error: Cannot resolve module 'react/lib/HTMLDOMPropertyConfig' in MYPATH/node_modules/html-react-parser/lib
 @ ./~/html-react-parser/lib/attributes-to-props.js 6:28-70

ERROR in ./~/html-react-parser/lib/property-config.js
Module not found: Error: Cannot resolve module 'react/lib/HTMLDOMPropertyConfig' in MYPATH/node_modules/html-react-parser/lib
 @ ./~/html-react-parser/lib/property-config.js 7:28-70

ERROR in ./~/html-react-parser/lib/property-config.js
Module not found: Error: Cannot resolve module 'react/lib/SVGDOMPropertyConfig' in MYPATH/node_modules/html-react-parser/lib
 @ ./~/html-react-parser/lib/property-config.js 8:27-68
````

--- a/lib/DOMPropertyConfig.js
+++ b/lib/DOMPropertyConfig.js
@@ -1,0 +1,25 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var HTMLDOMPropertyConfig;
+var SVGDOMPropertyConfig;
+
+// HTML and SVG DOM Property Config
+// moved to `react-dom` in v15.4.x
+try {
+    HTMLDOMPropertyConfig = require('react-dom/lib/HTMLDOMPropertyConfig');
+    SVGDOMPropertyConfig = require('react-dom/lib/SVGDOMPropertyConfig');
+} catch (error) {
+    HTMLDOMPropertyConfig = require('react/lib/HTMLDOMPropertyConfig');
+    SVGDOMPropertyConfig = require('react/lib/SVGDOMPropertyConfig');
+}
+
+/**
+ * Export config.
+ */
+module.exports = {
+    HTMLDOMPropertyConfig: HTMLDOMPropertyConfig,
+    SVGDOMPropertyConfig: SVGDOMPropertyConfig
+};

--- a/lib/attributes-to-props.js
+++ b/lib/attributes-to-props.js
@@ -3,7 +3,7 @@
 /**
  * Module dependencies.
  */
-var HTMLDOMPropertyConfig = require('react/lib/HTMLDOMPropertyConfig');
+var HTMLDOMPropertyConfig = require('./DOMPropertyConfig').HTMLDOMPropertyConfig;
 var utilities = require('./utilities');
 var propertyConfig = require('./property-config');
 

--- a/lib/property-config.js
+++ b/lib/property-config.js
@@ -4,8 +4,9 @@
  * Module dependencies.
  */
 var utilities = require('./utilities');
-var HTMLDOMPropertyConfig = require('react/lib/HTMLDOMPropertyConfig');
-var SVGDOMPropertyConfig = require('react/lib/SVGDOMPropertyConfig');
+var DOMPropertyConfig = require('./DOMPropertyConfig');
+var HTMLDOMPropertyConfig = DOMPropertyConfig.HTMLDOMPropertyConfig;
+var SVGDOMPropertyConfig = DOMPropertyConfig.SVGDOMPropertyConfig;
 
 var config = {
     html: {},

